src/web/app/pages-instructor/instructor-sessions-page/instructor-sessions-page.component.html

```diff
@@ -44,7 +44,7 @@
   <div class="row">
     <div class="col-12 text-muted">
       <p>
-        Note: The table above doesn't contain sessions from archived courses and soft-deleted courses.
+        Note: The table above doesn't contain sessions from archived courses and deleted courses.
```

Comment:
This doesn't blend well with the sentence right after it: it says "course in recycle bin" vs here where it says "deleted courses", and those two could (and actually do) mean different things.

Reply:
Hi, what do you suggest?

Reply:
Following the succeeding sentence: "courses in recycle bin"